moninag/static/src/ts/app/contacts/contact.ts

```diff
@@ -0,0 +1,14 @@
+export class Contact {
+
+    id: number;
+    email: string;
+    first_name: string;
```

Comment:
it is strongly suggested to use camelCase for property names and local variables. Here and elsewhere in this pull request.
(According Microsoft typescript style guide https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)